public/scripts/app.js

```diff
@@ -1,24 +1,18 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
-var $p_id;
+  var $profile_id;
 
-$("#addProject").fadeOut();
-$("#addVacation").fadeOut();
+  $("#addProject").fadeOut();
+  $("#addVacation").fadeOut();
 
-$('ul.nav li').click(function(){
-
-    console.log("event fired");
```

Comment:
Get rid of `console.log`s like this before you commit. Logs like this are just for help during development.